site/src/routes/_components/WhosUsingSvelte.svelte

```diff
@@ -90,5 +90,6 @@
 	<a target="_blank" rel="noopener" href="https://m.tokopedia.com"><img src="organisations/tokopedia.svg" alt="Tokopedia logo" loading="lazy"></a>
 	<a target="_blank" rel="noopener" href="https://webdesq.net"><img src="organisations/webdesq.svg" alt="Webdesq logo" loading="lazy"></a>
 	<a target="_blank" rel="noopener" href="https://zevvle.com/"><img src="organisations/zevvle.svg" alt="Zevvle logo" loading="lazy"></a>
+	<a target="_blank" rel="noopener" href="https://xmas.farmbox.ae/"><img src="organisations/farmbox.svg" alt="Farmbox logo" loading="lazy"></a>
```

Comment:
Any reason particular reason why the root domain isn't specified instead of the one targeted at holiday shopping?

Reply:
yep, because our mainsite is not using svelte (yet) but the mini site for our xmas offer is.

Reply:
but I can link to the main site if you prefer...

Reply:
Thanks!

Ok, that makes sense. I just wanted the links to stay relevant and it's not required that the site you link has Svelte on it. Some sites listed here only use it for intranet and internal stuff which the general public wouldn't see.

Hopefully, your trial run will wow the team and you'll be able to deploy on a larger scale!
Good luck with your development!